### PR TITLE
Foremost Bug Fix

### PR DESCRIPTION
**Pull request by zkkok**
# Changes

- Verbose and Quiet mode: Fixed the bug so both can't be selected at once
- Write All Headers Error: Unable to reproduce the error. Tool uses the correct parameter -a for "write all headers"
- White Screen Error: Unable to reproduce the error. Clicking on quiet and verbose modes does not trigger white screen

### CODE REVIEW

File: src/components/Foremost/Foremost.tsx [hunk 1]
```diff
@@ -172,18 +174,20 @@ const ForemostTool = () => {
                     placeholder={"Specify types (comma-separated) e.g., jpg,doc. if blank will retrieve all."}
                     {...form.getInputProps("types")}
                 />
-                <Stack spacing="lg">
-                    {/* Quiet Mode */}
-                    <Checkbox
-                        label={"Quiet Mode - enables quiet mode. Suppress output messages."}
-                        {...form.getInputProps("quiet" as keyof FormValuesType)}
+                {!checkedVerbose && (
```
Comment (bradleybogg) [inline src/components/Foremost/Foremost.tsx:177]: Consider using a single switch statement. This reduces code complexity.
I have tested the below for you as working, however, it requires you first rectify camelCase naming.

```
<Switch
    label={checkedQuiet ? "Verbose Mode" : "Quiet Mode"}
    checked={checkedQuiet ? checkedVerbose : !checkedVerbose}
    onChange={(e) => {
        if (checkedQuiet) {
            setCheckedVerbose(e.currentTarget.checked);
        } else {
            setCheckedQuiet(e.currentTarget.checked);
        }
    }}
/>
```

- [ ] Review code reccomendation.
- [ ] Implement as required.

**Comment by bradleybogg** [comment]
Hi @zkkok,

When creating your branch please use the convention we have specified. This is best completed by creating it from the github issue.

Kind Regards,

Bradley

File: src/components/Foremost/Foremost.tsx (hunk 1 above)
Comment (zkkok) [reply src/components/Foremost/Foremost.tsx:177]: The code is not suitable for the issue, it has a default mode also, without quiet and verbose flags. Single switch statements cannot handle three scenarios

File: src/components/Foremost/Foremost.tsx [hunk 2]
```diff
@@ -33,6 +33,8 @@ const ForemostTool = () => {
     // State hooks for loading, output, and advanced mode switch
     const [loading, setLoading] = useState(false);
     const [output, setOutput] = useState("");
+    const [checkedVerbose, setCheckedverbose] = useState(false);
```
[the remark this answers is not in the document]
Comment (zkkok) [reply src/components/Foremost/Foremost.tsx]: Corrected camelcasing

**Review by bradleybogg — CHANGES_REQUESTED** [review]
Requires application testing

- [x] @gangamarx please conduct application testing.

Thank you

**Comment by gangamarx** [comment]
Hi @zkkok ,
As requested, I ran a full test, and I can confirm all test completed successfully.
Verbose and Quiet mode: Can confirm that only can be selected.
Write All Headers Error: No errors upon checking
White Screen Error: Ran the test on different *.dd files, produced no white screen.
tested-successfully

**Review by bradleybogg — APPROVED** [review]

**Pull request merged**